lektor/admin/static/js/dialogSystem.tsx

```diff
@@ -1,19 +1,35 @@
 import hub from "./hub";
 import { DialogChangedEvent } from "./events";
+import FindFiles from "./dialogs/findFiles";
+import ErrorDialog from "./dialogs/errorDialog";
+import Publish from "./dialogs/publish";
+import Refresh from "./dialogs/Refresh";
+
+export type Dialog =
```

Comment:
Isn't `typeof` only useful on instances? Aren't these types already?

Reply:
Classes can be used as types directly, yes, but as a type a class refers to the type of the class instance. Since we're using `Dialog` where we expect the class and not a class instance, we use `typeof` to obtain the type of the class itself.